specs/experimental/plasma.md

```diff
@@ -67,6 +67,16 @@ The [batching][batcher] and compression of input data remain unchanged. When a b
 to be submitted to the inbox address, the data is uploaded to the DA storage layer instead, and a
 commitment (keccak256 hash) is submitted as the bacher inbox transaction call data.
 
+Commitment txdata introduces version `1` to the [transaction format](derivation.md#batcher-transaction-format),
+in order to interpret the txdata as a commitment during the l1 retrieval step of the derivation pipeline:
+
+| `version_byte` | `tx_data`   |
```

Comment:
Could we add this to the other page & note it as experimental rather than only specifying it here.